controller/search_blackbox_test.go

```diff
@@ -930,3 +930,83 @@ func (s *searchBlackBoxTest) TestIncludedParents() {
 	}
 	assert.Equal(s.T(), successCnt, 4)
 }
+
+func (s *searchBlackBoxTest) TestFilterAssigneeNullAfterWIUpdate() {
```

Comment:
Please add some comments suggesting what this test is about

Reply:
I would prefer to rewrite the test method name instead of writing a comment. But in this case, this is self explanatory.

Reply:
I would organize tests like these differently with sub-tests that act as your context. Without reading much of your test I think you're doing this here:

You update a workitem (1st context) by removing assignees (2nd context) and then you double check that the assignees value is null (3rd context).

In Go I would write a test like this:

```go
func TestUpdateWorkItem(t *testing.T) {
    t.Run("assignees", func(t *testing.T) {
        t.Run("remove", func(t *testing.T) {
            t.Run("ok", func(t *testing.T) {
                // double check that assignees is null
            })
        })
    })
}
```

You will then end up with a much more descriptive test name: `TestUpdateWorkItem/assignees/remove/ok`. It is more descriptive because you hide the implementation detail and promote the fact that the removal of assignee works. In your test expose the implementation detail by saying `...AssigneeNull...`. That doesn't tell me anything as somebody who hasn't seen your code before.

@xcoulon please, let us know how you would organize a test like this?

Reply:
Fixed. 9392cc941712846c91f70d9a694da3a5c30acb4a